frontend/pages/hosts/details/cards/Policies/HostPolicies.tsx

```diff
@@ -18,13 +20,15 @@ interface IPoliciesProps {
   isLoading: boolean;
   deviceUser?: boolean;
   togglePolicyDetailsModal: (policy: IHostPolicy) => void;
+  hostPlatform: string;
```

Comment:
maybe we can add the new options to and use the `SupportedPlatform` type in the future? I know the typing around platforms is hairy so non-blocking.

Reply:
Great suggestion--Lots of cobwebs uncovered, we can definitely improve types around platforms.